src/magicbox/SuggestionsManager.ts

```diff
@@ -65,16 +104,15 @@ export class SuggestionsManager {
     });
 
     this.suggestionsListbox = this.buildSuggestionsContainer();
-    this.suggestionsPreviewContainer = this.initPreviewForSuggestions(this.suggestionsListbox);
-    $$(this.element).append(this.suggestionsPreviewContainer.el);
+    $$(this.element).append(this.suggestionsListbox.el);
     this.addAccessibilityPropertiesForCombobox();
     this.appendEmptySuggestionOption();
   }
 
   public handleMouseOver(e) {
     let target = $$(<HTMLElement>e.target);
-    let parents = target.parents(this.options.selectableClass);
-    if (target.hasClass(this.options.selectableClass)) {
+    let parents = target.parents(this.options.suggestionClass);
```

Comment:
parent can be changed into a const, might be applicable to other let variables of this file.

Reply:
I didn't notice it, I'm changing them all to `const`. I'm also installing `eslint` locally for the sake of finding those faster in the future.